test/E2ETests/TestAssets/DeploymentTestProject/DeploymentTestProject.csproj

```diff
@@ -4,15 +4,15 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFramework>net462</TargetFramework>
     <IsPackable>false</IsPackable>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputPath>$(RepoRoot)artifacts\TestAssets\DesktopDeployment\$(DependencyCopyBehavior)\</OutputPath>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(RepoRoot)src\TestFramework\Extension.Desktop\Extension.Desktop.csproj" />
-    <ProjectReference Include="$(RepoRoot)src\TestFramework\MSTest.Core\MSTest.Core.csproj" />
+    <ProjectReference Include="$(RepoRoot)src\Adapter\MSTest.CoreAdapter\MSTest.CoreAdapter.csproj" />
```

Comment:
I think we should try without this change. It feels like it won't be required with the other changes we did.